crates/hyperqueue/src/bin/hq.rs

```diff
@@ -111,6 +112,7 @@ enum SubCommand {
     /// Automatic allocation management
     #[clap(name = "alloc")]
     AutoAlloc(AutoAllocOpts),
+    Event(EventOpts),
```

Comment:
Maybe call it ``Events`` or ``EventLog`` ? (as you are working with a more than one event